schema.graphql

```diff
@@ -493,3 +493,33 @@ type StakingDailySnapshot @entity {
   totalStaked: BigDecimal!
   totalStakedUSD: BigDecimal!
 }
+
+type RewardClaim @entity {
```

Comment:
RewardClaim is unique per tx_hash - meaning we only create a single RewardClaim for all the events we need to handle in a given tx.